src/components/aside/Aside.js

```diff
@@ -1,7 +1,63 @@
 import React from 'react';
 
+import { Link } from 'react-router-dom';
+import './Aside.scss';
+
 const Aside = () => {
-  return <div>무지막지</div>;
+  return (
+    <div className="aside">
+      {CATEGOTY_LIST.map(category => {
```

Comment:
```suggestion
      {CATEGOTY_LIST.map(({id, name, type}) => {
```

구조분해할당 해주면 훨씬 깔끔하겠네요! 함수 매개변수자리에서도 바로 구조분해할당 할 수 있습니다!

Reply:
aside 부분은 다른 팀원이 변경할 예정이여서 이 부분은 다른팀원이 수정 후 올리겠습니다!